package.json

```diff
@@ -25,10 +23,11 @@
     "lint": "eslint .",
     "prepublishOnly": "tap --no-check-coverage test/internals/version.test.js",
     "test": "npm run lint && npm run transpile && tap --ts && jest test/jest && npm run test-types",
-    "test-ci": "npm run lint && npm run transpile && tap --ts --no-check-coverage --coverage-report=lcovonly && npm run test-types",
+    "test-ci": "npm run lint && npm run transpile && tap --ts --no-check-coverage --coverage-report=lcovonly && npm run test-types && npx midnight-smoker test-ci-smoke",
```

Comment:
no additional dependency is needed?

Reply:
Nope

Reply:
I would prefer to avoid `npx` and add `midnight-smoker` as a devdep.

Reply:
@mcollina, added `midnight-smoker` as dev-dep. Included separate script as well